src/gpu/nufft_synthesis.cpp

```diff
@@ -249,8 +253,12 @@ auto NufftSynthesis<T>::get(View<T, 2> out) -> void {
 
   DeviceAccessor<T, 2> outDevice(queue, out);
 
-  const T scale =
-      totalCollectCount_ ? static_cast<T>(1.0 / static_cast<double>(totalCollectCount_)) : 0;
+  const T visScale =
```

Comment:
I'd suggest to add an option similar to `normalizeImage` in `NufftSynthesisOptions` and `StandardSynthesisOptions ` for the scaling based on number of visibilities.

Reply:
Done. On the Python side this can be disabled with:
`opt.set_normalize_image_by_nvis(False)`